src/lowlevel/Video.cpp

```diff
@@ -896,6 +896,35 @@ Rectangle Video::get_viewport() {
 
 /**
  * \brief Converts window coordinates to quest size coordinates.
```

Comment:
Ok, then can your remove the previous version of the function? Yours is better. It looks like no one checked the boolean result anyway.